pyscal/pyscalcli.py

```diff
@@ -165,8 +165,8 @@ def main():
             slgof=args.slgof,
             family2=args.family2,
         )
-    except ValueError:
-        # If ValueErrors, error messages have already been printed
+    except ValueError as err:
+        print(str(err))
```

Comment:
Why bother to `try-except` + `print(str(err))` and not just let `pyscal_main` crash it and throw the error?

Reply:
If you have control over all error situations (hah!) it is better (??) not to burden end-users with a full trace-back (then you get back that there is something wrong with komodo, because they see that string printed all over the screen). 

But this is absolutely up for discussion

Reply:
On the other hand: often a lot easier to help out with debugging if the traceback is available, unless all possibly thrown errors are very specific, and on point, in all error situations (hah! 😉)

Reply:
Exactly. No perfect solution. The pyscal client already has a `--debug` option, print traceback if that is activated?

Reply:
sounds good 😊 then the traceback is at least accessible 👍